### PR TITLE
Jest pup tests

### PR DESCRIPTION
**Pull request by andrewdeanyoder**
Hey Matt, would you mind taking a look when you get to your review time?

### CODE REVIEW

File: client/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    };
+  }
+
+  componentDidMount() {
+    axios.get(`/images/${this.state.productId}`)
+      .then((pictures) => {
+        this.setState({
+          images: pictures.data,
+        });
+        return pictures.data;
```
Comment (Matt-Wigg) [inline client/components/App.jsx:22]: I don't think you need this return statement. Where are you returning it to?

File: tests/app.test.js [hunk 2]
```diff
@@ -10,3 +19,22 @@ describe('Main Carousel', () => {
...
+        fullSizeURL: 'http://www.google.com',
+        thumbnailURL: 'http://someURL.com',
+      },
+    ];
+    const response = {data: fakeImage};
+    axios.get.mockResolvedValue(response);
+
+    return App.componentDidMount().then((data) => expect(data).toEqual(fakeImage));
+  });
+});
```
Comment (Matt-Wigg) [inline tests/app.test.js:40]: nice!!!

File: tests/seed.test.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    });
+  });
+  // use the results in each of the tests
+
+  test('does not assign productIds greater than 99 or less than 0', () => {
+    // iterates through all records
+    allRecords.forEach((record) => {
+      // checks that productId is in the acceptable range
+      expect(record.productId).toBeLessThan(100);
+      expect(record.productId).toBeGreaterThan(-1);
```
Comment (Matt-Wigg) [inline tests/seed.test.js:32]: This works, but do you want to be consistent with your use of `.toBeGreaterThanOrEqual()`?

File: client/components/App.jsx (hunk 1 above)
Comment (andrewdeanyoder) [reply client/components/App.jsx:22]: True. this was an experiment to try and make the tests work. I'll take it out.

File: tests/app.test.js (hunk 2 above)
Comment (andrewdeanyoder) [reply tests/app.test.js:40]: It doesn't completely work yet...

File: tests/seed.test.js (hunk 3 above)
Comment (andrewdeanyoder) [reply tests/seed.test.js:32]: Good catch.

**Pull request merged**